bats/tests/extensions/containers.bats

```diff
@@ -59,7 +59,7 @@ encoded_id() { # variant
 @test 'image - install' {
     rdctl api --method=POST "/v1/extensions/install?id=$(id vm-image)"
 
-    run rdctl api /v1/extensions
+    run --separate-stderr rdctl api /v1/extensions
     assert_success
```

Comment:
Under which circumstances can `rdctl api` print to `STDERR` but still exit with status `0`? That sounds like a bug to me and should be fixed instead of the tests ignoring it.

The only code I see writing explicitly to `os.Stderr` is in `displayAPICallResult`, but the code will either return a non-nil error (so Cobra should exit with non-0 exit code), or it calls `os.Exit(1)` directly.

I wonder if we should have a helper like this:

```bash
run_without_error() {
    run --separate-stderr "$@"
    assert_success
    # Verify that $stderr is empty, but print it and fail when it isn't
    output=$stderr assert_output
}
```

And then verify that the function succeeds and writes nothing to `STDERR` with:

```suggestion
    run_without_error rdctl api /v1/extensions
```